src/testrel.jl

```diff
@@ -686,7 +686,7 @@ function _test_rel_step(
                 @debug("$name: Expected problem", problem)
             else
                 unexpected_errors_found |= in(problem[:severity], error_levels)
-                @info("$name: Unexpected problem: $(problem[:code])")
```

Comment:
This could be prettier. The result is something like:
`Unexpected problem: Dict{Symbol, Any}(:line => 1, :code => "PARSE_ERROR", :severity => "error")`